cyclops-ctrl/internal/handler/handler.go

```diff
@@ -96,6 +97,8 @@ func (h *Handler) Start() error {
 	h.router.GET("/resources/pods/:namespace/:name/:container/logs", modulesController.GetLogs)
 	h.router.GET("/resources/pods/:namespace/:name/:container/logs/stream", sse.HeadersMiddleware(), modulesController.GetLogsStream)
 	h.router.GET("/resources/pods/:namespace/:name/:container/logs/download", modulesController.DownloadLogs)
+	h.router.GET("/resources/deployments/:namespace/:deployment/:container/logs", modulesController.GetDeploymentLogs)
+	h.router.GET("/resources/statefulsets/:namespace/:name/:container/logs", modulesController.GetStatefulSetsLogs)
```

Comment:
Im not sure these two methods work. In the case of multiple pods for a deployment,  it will just append logs from one pod to another regardless of the time those were produced at. The logs should be sorted by when they have been produced. Could you also check that out?

Also, we could just have a single endpoint that would be provided with a `group`, `version`, `kind`, `name` and `namespace` and it could just return all the logs for the requested resource. That way we don't have to duplicate endpoints for Deployments, Statefulsets, Pods...